v0/destinations/posthog/data/PHIdentifyConfig.json

```diff
@@ -0,0 +1,20 @@
+[
+  {
+    "destKey": "distinct_id",
+    "sourceKeys": [
+      "userId",
+      "context.traits.userId",
```

Comment:
Need to give precedence to `traits.userId` and `traits.id`

Reply:
same as above: 
"userId": [
    "userId",
    "traits.userId",
    "traits.id",
    "context.traits.userId",
    "context.traits.id",
    "anonymousId"
  ],

Reply:
Done